bundler/lib/bundler/lockfile_parser.rb

```diff
@@ -21,6 +44,7 @@ class LockfileParser
       Gem::Version.create("1.10") => [BUNDLED].freeze,
       Gem::Version.create("1.12") => [RUBY].freeze,
       Gem::Version.create("1.13") => [PLUGIN].freeze,
+      Gem::Version.create("2.5.0") => [CHECKSUMS].freeze,
```

Comment:
It makes sense to ship this with 2.5.0.

What happens to existing lockfiles not including this section? Are we keeping the current behavior in that case, and limiting this to fresh generated lockfiles, or this will fill in the checksums when Bundler 2.5 is run on them?

Reply:
Same as any of the other sections that have been added in this list -- bundler will not add the section if run with `--frozen` or `--deployment`, or if not unlocking anything